test/mocha/10-api.js

```diff
@@ -1,6 +1,204 @@
+      const {response, error, data} = await api.post({path});
+      //FIXME this is a client error where the wrong service was
+      // requested and should probably return 404.
+      const expected = {status: 500};
+      shouldError({response, error, data, path, expected});
+    });
+    it('should error if service doesn\'t have an initial step.', async () => {
+      const path = 'exchanges/noInitialStep';
+      const {response, error, data} = await api.post({path});
+      const expected = {status: 500};
+      shouldError({response, error, data, path, expected});
+    });
+    it('should error if in interact a service has an invalid type.',
+      async () => {
+        const path = 'exchanges/invalidServiceType';
+        const {response, error, data} = await api.post({path});
+        const expected = {status: 500};
+        shouldError({response, error, data, path, expected});
+      });
+  });
+  describe(`${exchange}/:transactionId`, () => {
+    it('should return a service with a transactionId', async () => {
+      const path = 'exchanges/multiStepUnmediated';
+      const initialResponse = await api.post({path});
+      shouldNotError({...initialResponse, path});
+      const {
+        verifiablePresentationRequest
+      } = testExchanges.oneStep.steps.initial;
+      shouldHaveVpWithTransactionId({
+        data: initialResponse.data,
+        path,
+        verifiablePresentationRequest
+      });
+      const {interact} = initialResponse.data.verifiablePresentationRequest;
+      shouldHaveInteractService({interact});
+      const [interactService] = interact.service;
+      const interactResponse = await api.put({
+        path: interactService.serviceEndpoint,
+        json: presentations.one
+      });
+      shouldNotError({
+        ...interactResponse,
+        path: interactService.serviceEndpoint
+      });
+    });
+    it('should error if transactionId is not found', async () => {
+      const path = 'exchanges/multiStepUnmediated';
+      const initialResponse = await api.post({path});
+      shouldNotError({...initialResponse, path});
+      const {
+        verifiablePresentationRequest
+      } = testExchanges.oneStep.steps.initial;
+      shouldHaveVpWithTransactionId({
+        data: initialResponse.data,
+        path,
+        verifiablePresentationRequest
+      });
+      const {interact} = initialResponse.data.verifiablePresentationRequest;
+      shouldHaveInteractService({interact});
+      const [interactService] = interact.service;
+      const interactResponse = await api.put(
+        {path: interactService.serviceEndpoint + 'notFound'});
+      //FIXME this should probably be 404 as the user is requesting
+      // an incorrect instanceId
+      shouldError({
+        ...interactResponse,
+        path: interactService.serviceEndpoint,
+        expected: {
+          status: 500
+        }
+      });
+    });
+
+  });
+  describe(`${exchangeInstance}/:stepId`, () => {
```

Comment:
Clean up style.

Reply:
what is the clean up here? Arrow functions in mocha tests? No space between `describe` statements?

Reply:
There's random whitespace to remove. And make sure if there's no whitespace before `describe` that's consistent throughout the file.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-vc-exchanger/pull/10/commits/420027fb6d9a1bdaf1ebb8acdad8d953f56e260d